### PR TITLE
Fix unit test because sproxyd-client had a small API change

### PR DESCRIPTION
**Pull request by JordanP**
https://github.com/scality/scality-sproxyd-client/commit/11ac4d9a58dbf4ac7ce3e2fff780c31155163f5f
removed the sproxyd_urls_set attribute.

Also, slightly increase the code coverage with a little change to
test_add_with_response_500

### CODE REVIEW

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/29/
Test PASSed.

File: test/unit/test_store.py [hunk 1]
```diff
@@ -84,7 +84,13 @@ def test_init(self):
         self.set_sproxyd_endpoints_in_conf(endpoints)
 
         store = Store(self.conf)
-        self.assertEqual(endpoints, store._sproxyd_client.sproxyd_urls_set)
+
+        actual_endpoints = set()
```
Comment (NicolasT) [inline test/unit/test_store.py]:
``` python
actual_endpoints = frozenset(store._sproxydd_client.get_next_endpoint() for _ in endpoints)
```

File: test/unit/test_store.py [hunk 2]
```diff
@@ -266,7 +272,8 @@ def test_add_with_response_412(self, mock_get_http_conn_for_put):
                 'get_http_conn_for_put', return_value=(mock.Mock(),
                                                        mock.Mock()))
     @mock.patch('scality_sproxyd_client.sproxyd_client.SproxydClient.'
-                'del_object')
+                'del_object', side_effect=scality_sproxyd_client.exceptions.
```
Comment (NicolasT) [inline test/unit/test_store.py]: Move to next line to get rid of trailing `.`?

**Comment by scality** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://37.187.159.67:5443/job/scality-glance-store-tox/30/
Test PASSed.

**Comment by NicolasT** [comment]
LGTM

**Pull request merged**